_posts/0000-01-02-SadNTasteless.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![perfectly-working-pc-windows-update-coming-to-hit-you-with-a-chair-the-simpsons](https://user-images.githubusercontent.com/73854135/97924429-41cecf00-1d25-11eb-8113-252c337705ee.jpg)

Reply:
hi